src/__tests__/ShallowWrapper-spec.js

```diff
@@ -484,6 +484,53 @@ describe('shallow', () => {
       expect(spy.args[3][0].hasClass('bux')).to.equal(true);
     });
 
+    describeIf(!REACT013, 'stateless functional components', () => {
+      it('finds nodes', () => {
```

Comment:
the first "it" block is just a sanity check (and passed prior to my change) - the second one is the one that changed in this PR (and failed prior to my change)

same in the ReactWrapper specs